app/importers/sharepoint_trade_article_data.rb

```diff
@@ -44,24 +43,38 @@ class SharepointTradeArticleData
     trade_url:             [],
   }.freeze
 
-  def initialize(resource = ENDPOINT)
-    @resource = resource
+  def initialize(s3 = nil)
+    @s3 = s3 || Aws::S3::Client.new(
+      access_key_id: ENV['AWS_ACCESS_KEY_ID_ARTICLES'], 
+      secret_access_key: ENV['AWS_SECRET_ACCESS_KEY_ARTICLES'])
```

Comment:
@tmhammer can you send me what these environment variables should be set to? They'll be needed for staging/production.